docker/Dockerfile

```diff
@@ -18,14 +18,22 @@ ENV CC_aarch64_unknown_linux_musl=clang
 ENV AR_aarch64_unknown_linux_musl=llvm-ar
 ENV CARGO_TARGET_AARCH64_UNKNOWN_LINUX_MUSL_RUSTFLAGS="-Clink-self-contained=yes -Clinker=rust-lld"
 
-FROM builder-${TARGETARCH} as builder
-RUN echo "Building for architecture:  $TARGET_RUST_ARCH"
+FROM builder-${TARGETARCH} AS builder
```

Comment:
nit: generally prefer to have formatting fixes in their own commit

That way, if we ever do have a bug that we need to bisect there's less code in the guilty commit to go through.